### DISCUSSION TITLE
Github Graduation Custom Trading Card Shipment 2022

### DISCUSSION THREAD

**Discussion by amandp13** [community/community#37629] [GitHub Education]
Hi,
I hope all are doing well.
It's almost six months since I received an email on May 5th that a trading card would arrive at my house and I still didn't receive the letter and custom trading card. Can someone help me with this?
Thank you.

**Comment by samus-aran** [comment]
Hi @amandp13 👋 

First off congratulations on your past graduation 🎉 We're sorry to hear that you hadn't received the GitHub Graduation swag from May. The Education team were aware that some graduates did not receive their trading card. To accommodate this we requested any graduates to [re-submit their details](https://github.com/education/GitHubGraduation-2022/issues/8395) before **October 7th 2022** to receive a re-printing of swag. Unfortunately this form is now closed. This may not be the response you had hoped for but we appreciate you taking part in the GitHub Graduation 2022!

**Reply by amandp13** [reply]
> Hi @amandp13 👋
> 
> First off congratulations on your past graduation 🎉 We're sorry to hear that you hadn't received the GitHub Graduation swag from May. The Education team were aware that some graduates did not receive their trading card. To accommodate this we requested any graduates to [re-submit their details](https://github.com/education/GitHubGraduation-2022/issues/8395) before **October 7th 2022** to receive a re-printing of swag. Unfortunately this form is now closed. This may not be the response you had hoped for but we appreciate you taking part in the GitHub Graduation 2022!

Thank you for the wish.
This is certainly the opposite response I had hoped for. Very disappointing.
